reward/stakingInfo.go

```diff
@@ -0,0 +1,152 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package reward
+
+import (
+	"github.com/klaytn/klaytn/blockchain"
+	"github.com/klaytn/klaytn/common"
+	"github.com/klaytn/klaytn/params"
+	"math"
+	"math/big"
+	"sort"
+)
+
+const (
+	AddrNotFoundInCouncilNodes = -1
+	maxStakingLimit            = uint64(100000000000)
+	DefaultGiniCoefficient     = -1.0
+)
+
+var (
+	maxStakingLimitBigInt = big.NewInt(0).SetUint64(maxStakingLimit)
+)
+
+// StakingInfo contains staking information.
+type StakingInfo struct {
```

Comment:
Is `StackInfo` struct exposed to external? If not, we can a lower case, `stakingInfo`.

Reply:
yes it is. `stakingInfo` is used in other package